src/ess/reduce/nexus/generic_types.py

```diff
@@ -156,6 +156,13 @@ class MonitorData(
 class Filename(sciline.Scope[RunType, Path], Path): ...
 
 
+@dataclass
+class PulseSelection(Generic[RunType]):
+    """Range of neutron pulses to load from NXevent_data groups."""
+
+    value: slice
```

Comment:
Is this the correct type? The code in the PR passes an empty tuple. So should this be something like ScipnnNexus' indexing type?

Reply:
Good catch, I changed code to pass a slice. I am not sure why an empty slice even worked with ScippNexus, since the value `PulseSelection` is put into a `dict` for the value of `event_time_zero`.